web/src/message_list_hover.js

```diff
@@ -42,7 +42,7 @@ export function message_hover($message_row) {
 
     const can_move_message = message_edit.can_move_message(message);
     const args = {
-        is_content_editable: is_content_editable && !message.status_message,
+        is_content_editable: is_content_editable && !message.is_me_message,
```

Comment:
Hmm, I wonder if this check is correct though; why are we not just looking at `is_content_editable`? I tested and indeed this being always false was actually correct; so just deleted the chekc.

(I think the history was that the edit icon was in some spot that didn't work for `/me` messages)